test/evergreen.yml

```diff
@@ -1384,6 +1394,23 @@ tasks:
               ./t -1 -C "direct_io=[data]" -c ../../../test/format/CONFIG.stress
             done
 
+  - name: test-checkpoint
```

Comment:
In current evergreen.yml there's another task called `checkpoint-test`. It would be nice to have a more specific name for this new task to avoid being confused with the other. I vote for `test-checkpoint-filetypes` :)

Reply:
Would the convention be checkout-filetypes-test?

Reply:
I used "test-checkpoint" task name to match it with the existing jenkins job ("wiredtiger-test-checkpoint"). Renaming this task to "test-checkpoint-filetypes" and respective function to "test checkpoint parameters". Please suggest for better name if any. . :)

Reply:
Actually, `checkpoint-filetypes-test` sounds better to me.

Reply:
Renaming the task to "checkpoint-filetypes-test" and respective function to "checkpoint filetypes test params"